test/chunking-form/index.js

```diff
@@ -10,7 +10,7 @@ runTestSuiteWithSamples('chunking form', path.resolve(__dirname, 'samples'), (di
 		() => {
 			let bundle;
 
-			for (const format of FORMATS) {
+			for (const format of [...FORMATS, ...(config.additionalFormats || [])]) {
```

Comment:
I know you use this to allow `UMD` here but really I do not think it makes a lot of sense considering UMD just does not support code-splitting. Instead, use a regular form test for the non-code-splitting case.